Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/appconfig/ui/AppConfigTestFragmentViewModel.kt

```diff
@@ -11,12 +12,16 @@ import timber.log.Timber
 
 class AppConfigTestFragmentViewModel @AssistedInject constructor(
     dispatcherProvider: DispatcherProvider,
-    private val appConfigProvider: AppConfigProvider
+    private val appConfigProvider: AppConfigProvider,
+    private val testSettings: TestSettings
 ) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
 
     val currentConfig = appConfigProvider.currentConfig.asLiveData()
     val errorEvent = SingleLiveEvent<Exception>()
 
+    val deviceTimeCheckDisabled = testSettings.isDeviceTimeCheckDisabled.flow
```

Comment:
just for clarification: this is solely used for the test menu and has no impact on any other behaviour in the app - e.g. preventing key download/submission or alike?

Reply:
No, this affects the whole app. Enabling this will fake the existence of the `disable-time-check` feature flag in the AppConfig:

https://github.com/corona-warn-app/cwa-app-android/blob/bd2f7bfa69ad287e93010da469120b70a904e885/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CWAConfigMapper.kt#L35-L39

Reply:
thanks